media-fonts/ibm_plex/ibm_plex-0.5.3.recipe

```diff
@@ -0,0 +1,34 @@
+SUMMARY="The package of IBM’s new typeface, IBM Plex"
+DESCRIPTION="Thanks for your interest in IBM Plex, IBM’s new corporate type family. \
+We welcome your feedback and contributions to this project. \
+IBM Plex is open source, and should be used by IBMers for all typographical situations, \
+replacing Helvetica Neue, whenever possible."
```

Comment:
Please rewrap `DESCRIPTION` to respect the 80-char per line limit as in our [Guidelines][0]

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#general-tips

Reply:
Well, I'd say we can make some exceptions if the line break comes after a full stop, even if one or more words of the next line would have fit in the line we are "terminating" before the 80 char limit.
This allows e.g. changes to take place without having to rewrap eveything.

Reply:
Actually, either you want two paragraphs, and there should not be a \ after the dot. Or, it is a single paragraph, and there is no reason to put a line break here.

    This is a paragraph.
    This is another paragraph.

    This is a single paragraph. \
    It is broken in two lines.